script/rust_to_wasm.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2025 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import argparse
+import os
+from lib.util import execute_stdout
+
+parser = argparse.ArgumentParser(description='Compile Rust packages to WASM')
```

Comment:
is this script actually necessary? Can we use `gn_run_binary.py`?

Reply:
We can't, because of
```python
env['PATH'] = args.wasm_bindgen_path + os.pathsep + env['PATH']
```

Reply:
why aren't we just adding the path in config.js?

Reply:
Can you leave comments to explain and also general description/usage comments?